### PR TITLE
IGNITE-13510 Added snapshot status command to control.sh and JMX.

### PR DESCRIPTION
**Pull request by NSAmelchev**
Create snapshot operation is in progress:
```
Create snapshot operation is in progress.
Snapshot name: snapshot1
Operation ID: 0c636326-76f8-4918-b15c-1010d9c47a5f
Started at: Aug 17, 2022 8:02:27 PM
Duration: 00:00:00.734

Estimated operation progress:

Node ID                                 Processed, bytes    Total, bytes    Percent
a4d3f584-bd67-4550-a26a-afa3bd500004    120.0 KB            2.2 MB          5%     
ef3a27fd-86eb-4886-97e6-7b061f100001    16.6 MB             34.1 MB         48%    
ffd8a17e-1fa4-4671-aac6-e2a344700003    12.6 MB             34.4 MB         36%    
072d4b79-d4f9-472e-8b6a-d05c34f00002    132.0 KB            2.1 MB          6%     
210db306-0aa9-45ef-a0ca-5b7f61200000    16.7 MB             34.2 MB         48%    
```

Restore snapshot operation is in progress:
```
Restore snapshot operation is in progress.
Snapshot name: snapshot1
Operation ID: 96201080-b1d0-4f0c-80ef-5e046c1f62cf
Started at: Aug 17, 2022 8:02:30 PM
Duration: 00:00:00.170

Estimated operation progress:

Node ID                                 Processed, partitions    Total, partitions    Percent
a4d3f584-bd67-4550-a26a-afa3bd500004    9                        9                    100%   
ef3a27fd-86eb-4886-97e6-7b061f100001    11                       16                   68%    
ffd8a17e-1fa4-4671-aac6-e2a344700003    14                       16                   87%    
072d4b79-d4f9-472e-8b6a-d05c34f00002    11                       12                   91%    
210db306-0aa9-45ef-a0ca-5b7f61200000    12                       16                   75%    
```

There is no create or restore snapshot operation in progress: 
```
There is no create or restore snapshot operation in progress.
```

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/visor/snapshot/VisorSnapshotRestoreTask.java [hunk 1]
```diff
@@ -25,8 +25,11 @@
 
 /**
  * Visor snapshot restore task.
+ *
+ * @deprecated Use {@link VisorSnapshotStatusTask} instead.
  */
 @GridInternal
+@Deprecated
```
Comment (xtern) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/snapshot/VisorSnapshotRestoreTask.java]: I think we should deprecate only job now.

Comment (NSAmelchev) [reply modules/core/src/main/java/org/apache/ignite/internal/visor/snapshot/VisorSnapshotRestoreTask.java]: Fixed, thanks.

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/snapshot/SnapshotStatusCommand.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+        SystemViewCommand.printTable(titles, F.asList(STRING, NUMBER, NUMBER, NUMBER),
+                rows, log);
+
+        log.info(U.nl());
+    }
+
+    /** {@inheritDoc} */
+    @Override public void parseArguments(CommandArgIterator argIter) {
+        // No-op.
```
Comment (xtern) [inline modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/snapshot/SnapshotStatusCommand.java:105]:
```suggestion
        if (argIter.hasNextSubArg())
            throw new IllegalArgumentException("Unexpected argument: " + argIter.peekNextArg() + '.');
```

Comment (NSAmelchev) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/snapshot/SnapshotStatusCommand.java]: Fixed, thanks

**Review by xtern — APPROVED** [review]

**Pull request merged**